runtime/matchers.go

```diff
@@ -0,0 +1,39 @@
+package runtime
+
+import "strings"
+
+// EqualMatcher performs a case-sensitive equality match for request metadata keys
+func EqualMatcher(s string) ServeMuxOption {
```

Comment:
Thinking further, perhaps the signature should be ```func(string) (string, bool)``` to allow transformations in addition to simple predicate filtering.